src/averell/readers/disco3.py

```diff
@@ -7,129 +7,65 @@
-                    work_dict.update({"poem_title": stanza.text, })
-                else:
-                    stanza_dict.update({
-                        'stanza_number': stanza.attrib["n"],
-                        'stanza_type': stanza.attrib["type"],
-                    })
-                    line_list = []
-                    for line in stanza:
-                        line_dict = {}
-                        if line.get("met") is not None:
-                            line_text = re.sub(r"[\n ]+", " ",
-                                               "".join(line.itertext()))
-                            line_dict.update({
-                                'line_number': str(line_number),
-                                'line_text': line_text,
-                                'metrical_pattern': line.attrib["met"],
-                                'rhyme': line.attrib.get("rhyme", "None"),
-                            })
-
-                            stanza_text.append(line_text)
-                            if line_dict["rhyme"] != "None":
-                                stanza_rhyme_pattern.append(
-                                    line.attrib["rhyme"])
-                            line_number += 1
-                            line_list.append(line_dict)
-
-                    if not stanza_rhyme_pattern:
-                        stanza_rhyme_pattern = "None"
-                    else:
-                        stanza_rhyme_pattern = "-".join(stanza_rhyme_pattern)
-                    stanza_dict.update({
-                        'rhyme_pattern': stanza_rhyme_pattern,
-                        'stanza_text': "\n".join(stanza_text),
-                        'lines': line_list,
-                    })
-                    stanza_list.append(stanza_dict)
-
-            work_dict.update({'stanzas': stanza_list})
-            sonnet_list.append(work_dict)
-    # End line groups
-    if is_ensemble:
-        id_list.sort()
-        for sonnet in sonnet_list:
-            sonnet.update(
-                {"work_number": str(id_list.index(sonnet['poem_id']) + 1)})
-    for sonnet in sonnet_list:
-        sonnet_dict = {**ensemble_dict, **sonnet}
-        yield sonnet_dict
+            stanza_text.append(line_text)
+            line_number += 1
+        stanza_list.append({
+            "stanza_number": str(stanza_number + 1),
+            "stanza_type": line_group.attrib["type"],
+            "lines": line_list,
+            "stanza_text": "\n".join(stanza_text),
+        })
+    poem.update({"stanzas": stanza_list})
+    return poem
 
 
 def get_features(path):
     """
-
-    :param path: path of the corpora folder
+    Function to parse all corpus poems
+    :param path: Corpus path
+    :return: list of poem dicts
     """
-    for filename in (Path(path) / XML_PATH).rglob('*.xml'):
-        parse_xml(str(filename))
+    feature_list = []
+    for filename in (Path(path)).rglob('*/per-sonnet/*.xml'):
```

Comment:
Not sure if 
`*/per-sonnet/*.xml'`
would work on unix/windows systems, we should use a Path() like in the other paths

Reply:
Done